lib/modules/manager/ansible-galaxy/util.ts

```diff
@@ -1,7 +1,7 @@
 import { regEx } from '../../../util/regex';
 
 export const newBlockRegEx = /^\s*-\s*((\w+):\s*(.*))$/;
-export const blockLineRegEx = /^\s*((\w+):\s*(.*))$/;
+export const blockLineRegEx = /^\s*((\w+):\s*(\S+))\s*$/;
```

Comment:
why do we use regex for parsing yaml inside this manager?

@zharinov can we use your parser framework if default yaml parser doesn't work?

Reply:
Can I mark this as resolved?

Reply:
Can I mark this as resolved?